src/ui/components/modal/HelpModal.tsx

```diff
@@ -74,6 +74,40 @@ export const HelpModal = ({ isOpen, setIsOpen }: Omit<ModalProps, 'title'>) => {
             <ChevronRightIcon className="h-4 w-4 dark:text-gray-400 text-gray-500 group-hover:opacity-50" />
           </a>
         </li>
+        <li>
+          <a
+            className="group flex w-full justify-between items-center"
+            href="https://github.com/paritytech/contracts-ui/blob/master/FAUCETS.md"
+            target="_blank"
+            rel="noopener noreferrer"
+          >
+            <div className="flex py-4">
+              <div className="w-5 mr-2 relative" style={{ position: 'relative', top: 1 }}>
+                <svg
```

Comment:
A solid icon might look better here (the other ones in the modal are also solid)